lib/qos/qos.c

```diff
@@ -0,0 +1,335 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <sys/slist.h>
+
+#include "qos.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(qos, CONFIG_QOS_LOG_LEVEL);
+
+#define INVALID -1
+
+/* Lookup table for QoS backoff messages. */
```

Comment:
These are times for the backoff timer, right? Probably "times" or "timeouts" would be better.
```suggestion
/* Lookup table for QoS backoff times. */
```

Reply:
Agreed.